orchestrator/migrations/versions/schema/2023-12-06_048219045729_add_workflow_id_to_processes_table.py

```diff
@@ -0,0 +1,46 @@
+"""Add workflow_id to processes table.
+
+Revision ID: 048219045729
+Revises: da5c9f4cce1c
+Create Date: 2023-12-06 15:33:46.997517
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+from orchestrator import db
+
+# revision identifiers, used by Alembic.
+revision = "048219045729"
+down_revision = "da5c9f4cce1c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(
+        "DELETE FROM processes WHERE NOT EXISTS (SELECT 1 FROM workflows wf WHERE processes.workflow = wf.name);"
```

Comment:
not sure if we want to delete these processes...
We need atleast a confirmation so we don't let other people delete processes that have workflow names of deleted workflows without notice

Reply:
There isn't really a way to get confirmation in a migration. It just has to work on deployment.

A different option would be to fail the migration and have the user manually deal with the dangling processes. Not sure if that is desirable either.

Reply:
In practice there won't be many processes that will be deleted, however maybe just to be sure we don't throw away other peoples data, we should just fail the migration with a good warning and let people solve their dangling processes.